config/rollup.config.js

```diff
@@ -57,6 +59,21 @@ export default function makeConfig(commandOptions) {
         sourcemap: false
       }
     ]
+  });
+
+  const makeESM = () => ({
+    ...config,
+    // These packages do not have a browser-ready ESM build, so we'll just
+    // bundle and ship them
+    external: external.filter(
```

Comment:
Word of caution here again; we don’t want to point `package.json:module` at this if we internalise styled-components.

If someone ends up using Spectacle with CRA and/or custom components that use styled-components, then we’ll have it twice in the bundle and it’ll likely break

Reply:
@kitten would you suggest making an ESM version with no external deps bundled  (that wouldn't work directly in the browser) and pointing `package.json:module` at that? Then we could create a separate 'browser' bundle with styled-components bundled and point `package.json:browser` at that?